packages/number-input/README.md

```diff
@@ -62,25 +62,25 @@ or
+| `description`     | `string`                                               | Text shown above the number input but below the label; gives more details about the requirements for the input.                                                                                              |                                     |
+| `value`           | `string`                                               | The controlled value of the number input.                                                                                                                                                                    |                                     |
+| `disabled`        | `boolean`                                              | Disables the component.                                                                                                                                                                                      | `false`                             |
+| `placeholder`     | `string`                                               | The placeholder text shown in the input field before the user begins typing.                                                                                                                                 |                                     |
+| `size`            | `'xsmall'` \| `'small'` \| `'default'` \| `'large'`    | Determines the size of the input.                                                                                                                                                                            | `default`                           |
+| `state`           | `'none'`\| `'error'`                                   | Describes the state of the TextInput element before and after the input has been validated                                                                                                                   | `'none'`                            |
+| `errorMessage`    | `string`                                               | Error message that appears below the input. Renders only if `state='error'`.                                                                                                                                 | `'This input needs your attention'` |
+| `successMessage`  | `string`                                               | Success message that appears below the input. Renders only if `state='valid'`.                                                                                                                               | `'Success'`                         |
+| `unit`            | `React.ReactNode`                                      | The content that appears to the right of the input if using a single unit. </br> </br>Required if using `unitOptions`. When using `unitOptions` this value becomes the controlled value of the select input. | `default`                           |
```

Comment:
Maybe `React.ReactNode | string` since it can't be a node when using the select input?

Reply:
Kk updated